core/src/main/java/com/netflix/conductor/metrics/Monitors.java

```diff
@@ -222,6 +221,11 @@ public static void recordWorkflowTermination(String workflowType, WorkflowStatus
             "" + ownerApp);
     }
 
+    public static void recordWorkflowStartSuccess(String workflowType, String version, String ownerApp) {
+        counter(classQualifier, "workflow_start_success", "workflowName", workflowType, "version", version, "ownerApp",
+            "" + ownerApp);
```

Comment:
This is tangential to the change.  The `"" + ownerApp` produces a String with the characters "null" if the value of `ownerApp` is `null`. Is that intention here? Do we want add a tag to the counter whose value is the String "null"? I see this pattern in some other methods in this class as well.

Reply:
changed to use `StringUtils.defaultIfBlank(ownerApp, "unknown")`